components/02-molecules/search-result/_yds-search-result.scss

```diff
@@ -0,0 +1,32 @@
+@use '../../01-atoms/atoms';
+@use '../../00-tokens/tokens';
+
+.search-results--summary {
+  font-style: italic;
+  color: var(--color-gray-600);
+  margin-bottom: var(--spacing-page-inner);
+}
+
+.search-form--page {
+  display: flex;
+  gap: var(--size-spacing-5);
+}
+
+.search-result {
+  @include atoms.clickable-component-heading-link;
+
+  padding: var(--size-spacing-5) 0;
+  border-bottom: 1px solid var(--color-gray-300);
+
+  .search-result__highlighted,
+  .search-result__teaser {
+    margin: var(--size-spacing-5) 0;
+  }
+
+  .search-result__teaser {
+    color: var(--color-gray-600);
+  }
```

Comment:
This is very minor, but I suggest moving these nested selectors outward.

So it reads more like this: 
 
```
.search-result {
  @include atoms.clickable-component-heading-link;

  padding: var(--size-spacing-5) 0;
  border-bottom: 1px solid var(--color-gray-300);

  @media (min-width: tokens.$break-mobile-max) {
    max-width: var(--size-component-layout-width-content);
  }
}

.search-result__highlighted,
.search-result__teaser {
  margin: var(--size-spacing-5) 0;
}

.search-result__teaser {
  color: var(--color-gray-600);
}
  ```

Reply:
Also, I notice that on hover the search result doesn't get a bottom box shadow like the design: https://www.figma.com/proto/G4sJ0BLu4jCJpGzW82JHE8/YaleSites-Components?node-id=6288%3A19237&scaling=min-zoom&page-id=6288%3A19236&hide-ui=1

I'll defer to Ashley on the design review here, but if we do want the bottom-only shadow on hover, there are 2 other instances in the codebase where this kind of hover box-shadow is in place. 

We have used the following (with the TODO comment in place): 
```
    &:hover {
      // @TODO: Temporary styles until we create a token
      // This shadow is hard coded because it has to be adjusted to make it
      // appear as though it's a shadow on only the border-bottom element.
      // There isn't currently anything else that looks like this. If there ever
      // Is, perhaps it should be captured as a token in a new drop-shadow set.
      // @LINK: https://yaleits.atlassian.net/browse/YALB-844.
      box-shadow: 0 8px 6px -6px hsl(0deg 0% 0% / 16%);
    }
```

Reply:
OK, @joetower, I fixed both of your comments above and added the @TODO so we know where to replace that once there is a token.

As for Ashley's comments below, I've fixed the last two on her list, but her comments about the breadcrumbs I would like to defer to you on this one. I'm happy to learn best practices on this, but from what Brian taught me, those breadcrumb changes shouldn't live in the search result sass, but instead in the breadcrumb sass - though they are only really used (for now) on the search result.

On line 11 of _yds-breadcrumbs.scss, Brain showed me that we could override breadcrumbs with no bottom margin that is being using on search results, but I'm not sure we want to use that same area for overriding the slash color and the mobile changes. Let me know your thoughts. Thanks!

Reply:
@marcb4k I would probably add: 
```
.search-result & {
    color:var(--color-basic-brown-gray);
  }
```
to the `03-organisms/menu/breadcrumbs/_yds-breadcrumbs.scss` file inside of the `.breadcrumbs__item {` codeblock, maybe at the bottom? 

```
.breadcrumbs__item {
  @include tokens.body-s;

  position: relative;
  flex-shrink: 0;
  display: flex;
  align-items: center;

  @media (max-width: tokens.$break-mobile-max) {
    [data-breadcrumbs-overflow='hidden'] & {
      display: none;
    }
  }

  &:not(:last-child) {
    margin-right: var(--size-spacing-5);
    padding-right: var(--size-spacing-5);

    &::after {
      content: '|';
      position: absolute;
      right: 0;
      transform: skew(-15deg) translateX(50%);
    }
  }

  .search-result & {
    color:var(--color-basic-brown-gray);
  }
}
```

This way it is all still contained in the same file. Looks like the `breadcrumb__item`s are using the standard body text color, while the links are `var(--color-basic-brown-gray);` so making the items the same color while in the search-result instance seems like a solid approach to me. 

Am I understanding the feedback correctly, @ahitson ?

Reply:
Yep!

Reply:
@ahitson - thanks for the feedback. I've made the changes and you can review here:

https://deploy-preview-176--dev-component-library-twig.netlify.app/?path=/story/molecules-search-result--search-result

As for the underline on hover, take a look at the custom card here, and add a lowercase "g" to the title and it happens there too - the hover line is below the "g".

https://deploy-preview-176--dev-component-library-twig.netlify.app/?path=/story/molecules-cards--custom-card

If we want to standardize on the line going through the "g", let us know. Thanks!